.github/workflows/ci.yml

```diff
@@ -272,3 +278,67 @@ jobs:
           files: |
             packages/*
             LICENSE
+  docker-image:
```

Comment:
can we only do this on `release`?

Reply:
As I understand it, master should be a dev-like branch in that project, and commits that are tagged exist to be releases, right? Am I understanding you correctly? Because I don't see a dev branch associated with it.

Reply:
> can we only do this on `release`?

A nightly release doesn't require a build image either?

Reply:
We can, but why?

Reply:
ok i might be a bit unclear - i saw this PR had a docker image built(that failed with no docker creds, which I'll add), which I don't think we need, before merging?